src/lib/transition_handler/catchup_scheduler.ml

```diff
@@ -225,7 +234,10 @@ module Make (Inputs : Inputs.S) = struct
           Hashtbl.set t.collected_transitions ~key:parent_hash
             ~data:(cached_transition :: cached_sibling_transitions) ;
           Hashtbl.update t.collected_transitions hash
-            ~f:(Option.value ~default:[])
+            ~f:(Option.value ~default:[]) ;
```

Comment:
Instead of `Hashtbl.update`, should we have Hashtbl.add_exn?

Reply:
No. If that transition has children, we don't want to change its list of children. If it doesn't have any child yet, then we create an entry for it.